src/tools/compiletest/src/runtest.rs

```diff
@@ -1135,6 +1135,7 @@ actual:\n\
             Some(ErrorKind::Error) => true,
             Some(ErrorKind::Warning) => true,
             Some(ErrorKind::Suggestion) => false,
+            Some(ErrorKind::Guess) => false,
```

Comment:
You have just added these annotations to many tests, so I thought they are mandatory now.